packages/pwa-builder-manager/src/editorAppMachine.ts

```diff
@@ -298,6 +305,7 @@ export function createEditorAppMachine(id: string) {
     [REDROP_FAILED]: [],
     [SELECT]: [],
     [SELECT_END]: [],
+    [DROPZONE_CREATION_IN_PROGRESS]: [],
```

Comment:
Add two subscribers - `DROPZONE_CREATED`  and `DROPZONE_DESTROYED`.

Reply:
Done